tests/periph/flashpage/main.c

```diff
@@ -428,7 +442,7 @@ static int cmd_test_last_raw(int argc, char **argv)
     flashpage_write(flashpage_addr(TEST_LAST_AVAILABLE_PAGE), raw_buf, sizeof(raw_buf));
 
     /* verify that previous write_raw effectively wrote the desired data */
-    if (memcmp(flashpage_addr(TEST_LAST_AVAILABLE_PAGE), raw_buf, strlen(raw_buf)) != 0) {
+    if (memcmp(flashpage_addr(TEST_LAST_AVAILABLE_PAGE), raw_buf, 16) != 0) {
```

Comment:
There was just a merge conflict here, due to the merge of #18093 3 weeks ago.
I resolved that and rebased.